messenger.go

```diff
@@ -61,6 +62,9 @@ type featureFlags struct {
 	// V1 messages adds additional wrapping
 	// which contains a signature and payload.
 	sendV1Messages bool
+
+	// datasync indicates whether messages should be sent using datasync, breaking change for non-v1 clients
```

Comment:
To clarify, this means exclusively datasync? As opposed to in addition. Can we add that in the comment?

Also, how does this impact e.g. public chats in terms of when this flag is set? Not clear from looking at this piece of the code.

Reply:
What's the relation between this flag and `sendV1Messages`? Do we want to support V1 messages with or without ds?

Reply:
The PR was merged without the clarification requested, did it fall through the cracks?

Reply:
I could not reply to the thread, no matter how much I refreshed the page, so I had to answer on a comment https://github.com/status-im/status-protocol-go/pull/34#issuecomment-515416417

Reply:
I was initially referring to Oskar's comment

Reply:
oh I see, yes, forgot to add that

Reply:
I will add in a separate pr

Reply:
Thanks!